hazelcast/src/main/java/com/hazelcast/query/impl/getters/ExtractorHelper.java

```diff
@@ -92,4 +92,16 @@ static String extractAttributeNameNameWithoutArguments(String attributeNameWithA
         }
     }
 
+    public static int indexOf(char[] input, char splitter, int offset) {
```

Comment:
we don't have commons-lang on the classpath and I want to control the implementation to be sure it's fast for the case.